test/build/base-url-homepage/package.json

```diff
@@ -10,6 +10,9 @@
     "scripts": {
       "mount:public": "mount public --to /",
       "mount:src": "mount src --to /_dist_"
+    },
+    "buildOptions": {
+      "optimize": false
```

Comment:
Here, and below, we added `buildOptions.optimize = false` to the tests. No output for them changed (save for the `env.js` minification)